### ISSUE TITLE
Sufia+PCDM Roadmap

### ISSUE THREAD

**Issue by mark-dce** [pbinkley/HC2015-program#69]
Presenter: Mike Giarlo (Mark e/m 6/26; Jon Stroop
Format: presentation (preping working group #11)

**Comment by hackartisan** [comment]
general-interest version of #62

**Comment by mark-dce** [comment]
Mark to talk with Jeremy about making #62 developer focused with #69 a general audience teaser for plenary

**Comment by mark-dce** [comment]
I talked with Mike G. and confirmed that he's willing to act as lead presenter (possibly with other's helping)

**Comment by mjgiarlo** [comment]
@mark-dce @richardGreen 

Hmmm, I thought I confirmed my availability and interest for this -- at least I don't see any outstanding email in my inbox about it.  I was the presenter in mind for this right?  If so, consider me confirmed to deliver this plenary talk.

There were a number of slots about Sufia and the PCDM.  If this is the talk that will partly cover ongoing work about PCDM and Sufia, I believe @jpstroop offered to co-present.

**Comment by jpstroop** [comment]
Yeah, if 'e/m' means email and I was supposed to receive one, I don't think I have. I'm happy to help on this one, if you'd like. Either that or I'd suggest making this _only_ about PCDM, and then move of the discussion about sufia/curation_concens etc. over to #11--I don't think that's a good idea, however, as this topic seems to be of interest to most of the community.

**Comment by mjgiarlo** [comment]
@jpstroop I sense that you're leaning towards the following:
- Cancel #11 
- Go forward with you and me co-presenting #69 and focusing on
  - **Brief** background info re: Sufia, Worthwhile, and PCDM to provide context to the audience
  - The state of PCDM dev in Hydra (the May & summer sprints, the new gem stack)
  - What's coming next (when will things be "done"?) and how can folks get involved

That's what I have in mind, too. How does that sound? I _suspect_ we can squeeze that into 20 minutes, though 25 or 30 would be better, @richardGreen @mark-dce etc. :wink:

**Comment by jpstroop** [comment]
:+1: I agree. I think that as long as we don't dally on PCDM for too long (which probably isn't appropriate anyway), we should be able to make it fit.

**Comment by mjgiarlo** [comment]
:+1: Limit that to a 30-second description and hand-wave in the direction of all the other PCDM program entries.  (I am also dreaming here of a snazzy, simple page like iiif.io but targeted at getting folks from 0 to 60 (well, maybe 30 :wink:) on PCDM  -- that way we can say, "just look at pcdm.io!" -- but I doubt any of us has time for such a thing.)

:cool: Thanks, @jpstroop
